civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +72,194 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .chr_panel--my-details__personal__contact_id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom: 0;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0;
+    position: absolute;
+    top: 0;
+
+    img {
+      border-radius: 50%;
+      height: 80px;
+      width: 80px;
+    }
+  }
+
+  .chr_panel__footer {
+    border-top: 0;
+    margin-top: 0;
+  }
+
+  .views-field-nothing {
+    .field-content {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      white-space: nowrap;
+      width: 100%;
+    }
+
+    .chr_panel--my-details__personal__contact_id,
+    .chr_panel--my-details__personal__gender,
+    .chr_panel--my-details__personal__marital_status {
+      margin-right: 1rem;
+    }
+  }
+
+  .views-field-ni-ssn-70 .field-content {
+    width: 100%;
+  }
+
+  .chr_panel--my-details__personal__nsi_exists {
+    &.chr_panel--my-details__personal__nsi_application_in_progress--1 {
```

Comment:
Looks like we are defining logic to show/hide elements using css, that is not ideal. View logic should be controlled from PHP/Drupal.

Reply:
Ok I will change the templates accordingly.